au.org.access-nri/model/experiment/reproducibility/checksums/1-0-0.json

```diff
@@ -1,8 +1,8 @@
 {
-    "$id": "https://raw.githubusercontent.com/ACCESS-NRI/schema/main/au.org.access-nri/model/access-om2/experiment/reproducibility/checksums/1-0-0.json",
+    "$id": "https://raw.githubusercontent.com/ACCESS-NRI/schema/main/au.org.access-nri/model/experiment/reproducibility/checksums/1-0-0.json",
```

Comment:
Is the `experiment` level necessary? If yes, then should the `au.org.access-nri/model/output` directory (containing the metadata schema) be moved into `experiment` since it's for "experiment" output.

Reply:
Yeah, I think `experiment` level might not be necessary, though I might be missing some of reasoning of why it was used in the first place. It was initially discussed here: https://github.com/ACCESS-NRI/schema/pull/5#issuecomment-1947093038 -  @aidanheerdegen do you have any thoughts on this?

Reply:
I'm not sure I can recreate my reasoning. I don't have a strong feeling, but I agree that there is an inconsistency that is undesirable so let's just get rid of `experiment` from the hierarchy for consistency and it seems a little redundant.

Reply:
Ok I've now removed the `experiment` level